src/library/columncache.cpp

```diff
@@ -181,7 +181,7 @@ void ColumnCache::slotSetKeySortOrder(double notation) {
     // A custom COLLATE function was tested, but using CASE ... WHEN was found to be faster
     // see GitHub PR#649
     // https://github.com/mixxxdj/mixxx/pull/649#discussion_r34863809
-    QString keySortSQL("CASE key_id ");
+    QString keySortSQL("CASE %1_id WHEN NULL THEN 0 ");
```

Comment:
https://github.com/Be-ing/mixxx/blob/key_sort/src/library/columncache.h#L114
columnSortForFieldIndex() uses QString::arg() which expects there to be a %1 in this string to replace with the column name